packages/appstore/src/components/wallet-transfer/timer/timer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { useCountdown } from '@deriv/hooks';
+import { Localize } from '@deriv/translations';
+
+type TTimerProps = {
+    className?: string;
+    from: number;
+    onComplete?: VoidFunction;
+};
+
+export type TTimerRef = {
+    reset: VoidFunction;
+};
+
+const Timer = React.forwardRef<TTimerRef, TTimerProps>(({ className, from, onComplete }, ref) => {
```

Comment:
Here i decided to use `useImperativeHandle` hook in order to be able to reset the `Timer` form the parent.

Reply:
I feel like this isn't very "React" like, right? Going back up the tree. 🤔 

It's still not great, but would just using the `key` prop to reset the timer be a bit clearer?

For example:

```ts
    useEffect(() => {
        reset();
    }, [resetKey]);
```

Then in the parent:

```ts
    const handleReset = () => {
        setTimerKey(prev => prev + 1);
    };
```

Reply:
Or maybe it would be better to call `useCountdown` in the parent, then pass the countdown into the component?

Reply:
hey @markwylde-deriv 
yeah i also thought about `key`, but applied `useImperativeHandle`
-----
the option with passing the whole countdown is really nice.
actually i tried to find the simplest way to reset the counter and seems like passing the whole countdown is good solution, will apply it

Reply:
Awesome @heorhi-deriv. It just feels a bit more "React"'y I think 🤔